x-pack/plugins/security_solution/public/common/store/data_table/epic_local_storage.ts

```diff
@@ -21,6 +21,7 @@ const {
   applyDeltaToColumnWidth,
   changeViewMode,
   removeColumn,
+  toggleDetailPanel,
```

Comment:
This was added as a bugfix. The flyout kept re-opening on refresh of the page after being closed. This was due to the fact that the opening and closing of the flyout wasn't tracked for updating the localstorage config.